.github/workflows/test.yml

```
@@ -35,296 +38,401 @@ concurrency:
         with:
-          name: coverage-report
-          path: coverage/
+          name: test-artifacts-coverage
+          path: |
+            coverage/
+            test-results/
           retention-days: 7
-
       - name: Coverage summary
         if: always()
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:30: Double quote to prevent globbing and word splitting

(shellcheck)